lib/rspec/core/bisect/example_minimizer.rb

```diff
@@ -8,7 +6,7 @@ module Bisect
       # repeatedly running different subsets of the suite.
       class ExampleMinimizer
         attr_reader :runner, :reporter, :all_example_ids, :failed_example_ids
-        attr_accessor :remaining_ids
+        attr_accessor :eliminated_ids
```

Comment:
Why the switch from `remaining_ids` to `eliminated_ids`?  They are just inverses (bearing the relationship `remaining_ids = non_failing_example_ids - eliminated_ids`) so it seems like whats possible or easy with one is just as possible or easy with the other, right?  I think tracking `remaining_ids` has a few advantages:

* It's what was already in place, so sticking with it means less churn here...which would help keep the diff on the PR focused on behavioral changes rather than including an equivalent structural change.
* `remaining_ids` seems more fundamental to me.  After all, that's what the final repo command is based on.  In addition, we keep passing `remaining_ids` as part of the notification events so it's clearly important externally.  Contrast that to `eliminated_ids` -- as far as I can see it's only used within this class.
* Having `remaining_ids` be a logical attribute that is computed on every call (e.g. for every notification that includes it) is going to be slower than if we tracked it directly.  (Admittedly, this perf difference is not going to be noticable.)

Thoughts?  If you still wanted to write code in terms of _eliminating_ ids, you could define an `eliminate_ids` helper method that would remove the ids from `remaining_ids`.